client/src/ui/explore/OpportunityCard.tsx

```diff
@@ -156,22 +151,23 @@ export function OpportunityCard({
             margin: 0,
           }}
         >
-          <Button
-            variant="contained"
-            style={{
-              backgroundColor: "#496FFF",
-              height: "40px",
-              width: "auto",
-              borderRadius: "20px",
-              boxShadow: "none",
-            }}
-            onClick={handleChatClick}
-          >
-            <FontAwesomeIcon icon={faComment} />
-            <Typography style={{ color: "white", marginLeft: ".5rem" }}>
-              {recentMessages}
-            </Typography>
-          </Button>
+          <Link href={`/message/company/${conversation_id}`}>
```

Comment:
Instead of using link, couldnt we use the handleChatClick Function to push to the message page? Having a link over a button seems redundant

Reply:
The router.push() function wasn't navigating to the page correctly which is why I resorted to using the Link component instead. We don't have to create the function to since Link handles it.

Reply:
Okay sounds good